src/uEMEP_auto_subgrid.f90

```diff
@@ -636,14 +637,14 @@ subroutine uEMEP_region_mask_new()
         ! Determine grid spacing and verify it is constant
         dx_regionmask = x_values_regionmask(2) - x_values_regionmask(1)
         do i = 2, nx_regionmask
-            if (.not. (x_values_regionmask(i) - x_values_regionmask(i-1) == dx_regionmask)) then
+            if (.not. (abs((x_values_regionmask(i) - x_values_regionmask(i-1) - dx_regionmask) / dx_regionmask) < epsilon0)) then
                 write(unit_logfile,'(A)') 'Not constant spacing in x coordinate in region mask'
                 stop
             end if
         end do
         dy_regionmask = y_values_regionmask(2) - y_values_regionmask(1)
         do i = 2, ny_regionmask
-            if (.not. (y_values_regionmask(i) - y_values_regionmask(i-1) == dy_regionmask)) then
+            if (.not. (abs((y_values_regionmask(i) - y_values_regionmask(i-1) - dy_regionmask) / dy_regionmask) < epsilon0)) then
```

Comment:
Maybe personal preference, but why use .not.? In my opinion, it is easier to read

if ((abs((y_values_regionmask(i) - y_values_regionmask(i-1) - dy_regionmask) / dy_regionmask) > epsilon0)) then

Reply:
OK, agree.